test/tests/resilience/pbs_hook_alarm_large_multinode_job.py

```diff
@@ -164,7 +164,7 @@ def test_epi_hook(self):
 
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'},
-                           jid, max_attempts=15, interval=2)
+                           jid, max_attempts=10, interval=2)
```

Comment:
shouldn't this be 20?

Reply:
Hi Prakash, this is max attempts when Job comes to R state. I have tested on slow machines and is working fine with 10 attempts. I have given interval as 2 so that's equivalent to 20 seconds.